Week-3/Exercise3.4/Exercise3.4.js

```diff
@@ -0,0 +1,18 @@
+function createStack() {
```

Comment:
The solution given is correct, simple & efficient. Also try implementing the solution using closures.